bundles/org.eclipse.swt/Eclipse SWT Tests/win32/org/eclipse/swt/widgets/WidgetWin32Tests.java

```diff
@@ -1,20 +1,16 @@
-/*******************************************************************************
- * Copyright (c) 2024 Yatta Solutions
- *
- * This program and the accompanying materials
- * are made available under the terms of the Eclipse Public License 2.0
- * which accompanies this distribution, and is available at
- * https://www.eclipse.org/legal/epl-2.0/
- *
- * SPDX-License-Identifier: EPL-2.0
- *******************************************************************************/
 package org.eclipse.swt.widgets;
```

Comment:
Copyright header is missing

Reply:
@fedejeanne Added copyright header and updated commit message as per the basics that you mentioned.

Reply:
fyi: the copyright header as it existed before would have been sufficient. There is no need for adding a "Contributors" section, as it is optional. Still, it is of course not forbidden to have/add one. I just do not see any benefit in having it.

Reply:
@HeikoKlare This copyright header is not part of this PR. But I can change it if you request so.

Reply:
@HeikoKlare I just added it as seen in other files but I could remove it if it's a problem

Reply:
The copyright header was already there, so there is no need to add something. You have only added the "Contributors" section.